core/src/main/java/fr/sncf/osrd/envelope/EnvelopeDebug.java

```diff
@@ -12,22 +12,6 @@
 
 @ExcludeFromGeneratedCodeCoverage
 public class EnvelopeDebug {
-    /** Export envelope as csv. */
-    public static void saveCSV(Envelope envelope, String path) {
-        try {
-            PrintWriter writer = new PrintWriter(path, StandardCharsets.UTF_8);
-            writer.println("position,speed");
-            for (var part : envelope) {
-                for (int i = 0; i < part.pointCount(); i++) {
-                    writer.println(String.format(Locale.US, "%f,%f", part.getPointPos(i), part.getPointSpeed(i)));
-                }
-            }
-            writer.close();
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
-    }
-
```

Comment:
This method is useful, why is it removed? It's incompatible with the changes?

Reply:
it is not used anymore, as envelopes can be displayed with the interactive swing UI